pkg/controller/helpers/helpers.go

```diff
@@ -104,3 +105,93 @@ func ReconcileDeployment(owner metav1.Object, deployment *appsv1.Deployment, req
 
 	return deployment, errorutil.WithMessagef(err, "error creating or updating Deployment %s/%s", deployment.Namespace, deployment.Name)
 }
+
+func ReconcileConfigMap(owner metav1.Object, configMap *corev1.ConfigMap, reqLogger logr.Logger,
```

Comment:
Is this taken from coredns operator on openshift?

Reply:
This is for the lighthouse coreDNS server , I created a minimal map for it. The update configmap have configs copied from Openshift.